buildpacks/scala/src/errors.rs

```diff
@@ -0,0 +1,274 @@
+                Please check the `{property_name}` property for quoting and escaping mistakes and try again.
+
+                Details: {error}
+            ", property_name = property_name, error = error }
+        ),
+
+        ScalaBuildpackError::CouldNotParseListConfigurationFromEnvironment(variable_name, error) => log_error(
+            format!("Invalid {variable_name} environment variable"),
+            formatdoc! {"
+                Could not parse the value of the {variable_name} environment variable into a list of words.
+                Please check {variable_name} for quoting and escaping mistakes and try again.
+
+                Details: {error}
+            ", variable_name = variable_name, error = error }
+        ),
+
+        ScalaBuildpackError::CouldNotParseBooleanFromProperty(property_name, error) => log_error(
+            format! ("Invalid {property_name} property"),
+            formatdoc! {"
+                Could not parse the value of the `{property_name}` property from the system.properties file into a 'true' or 'false' value.
+                Please check `{property_name}` for mistakes and try again.
+
+                Details: {error}
+            ", property_name = property_name, error = error }
+        ),
+
+        ScalaBuildpackError::CouldNotParseBooleanFromEnvironment(variable_name, error) => log_error(
+            format!("Invalid {variable_name} environment variable"),
+            formatdoc! {"
+                Could not parse the value of {variable_name} environment variable into a 'true' or 'false' value.
+                Please check {variable_name} for mistakes and try again.
+
+                Details: {error}
+            ", variable_name = variable_name, error = error }
+        ),
+
+        ScalaBuildpackError::CouldNotConvertEnvironmentValueIntoString(variable_name, value) => log_error(
+            format!("Invalid {variable_name} environment variable"),
+            formatdoc! {"
+                Could not convert the value of the environment variable {variable_name} into a string. Please
+                check that the value of {variable_name} only contains Unicode characters and try again.
+
+                Value: {value}
+            ", variable_name = variable_name, value = value.to_string_lossy() }
+        ),
+
+        ScalaBuildpackError::CouldNotParseListConfigurationFromSbtOptsFile(error) => log_error(
+            "Invalid .sbtopts file",
+            formatdoc! {"
+                Could not read the value of the .sbtopts file into a list of arguments. Please check
+                the file for mistakes and please try again.
+
+                Details: {error}
+            ", error = error }
+        ),
+
+        ScalaBuildpackError::MissingStageTask => log_error(
+            "Failed to run sbt!",
+            formatdoc! {"
+                It looks like your build.sbt does not have a valid 'stage' task. Please read our Dev Center article for
```

Comment:
Minor suggestion. Read as a verb mildly implies that they've not yet read it (as in the dreaded RTFM comment). Reference is longer but softer.

```suggestion
                It looks like your build.sbt does not have a valid 'stage' task. Please reference our Dev Center article for
```